database/backend/caching.json

```
@@ -1 +1,52 @@
-[]
\ No newline at end of file
+[
+    {
+        "name": "Aws caching overview",
+        "description": "In computing, a cache is a high-speed data storage layer that stores a subset of data. It's typically transient in nature so that future requests for said data are served up faster than is possible, which then accesses the data’s primary storage location. Caching also allows developers to efficiently reuse previously retrieved or computed data.",
```

[Proofreading] I think it would be more beneficial on how this source can be helpful instead of defining the actual term. 
```suggestion
        "description": "If you're looking to learn about Caching, check out this overview from Amazon",
```